src/os_win32.c

```diff
@@ -3556,13 +3556,15 @@ mch_init_c(void)
     static void
 mch_exit_c(int r)
 {
+    int fTermcapMode = g_fTermcapMode;  // Copy flag since stoptermcap() will clear the flag.
```

Comment:
This line exceeds 80 columns.

Reply:
I can fix this :)

Reply:
Sorry for the long line.  Is this a Vim coding style requirement?  I read develop.txt and did not see any mention of a maximum line length.  Did I miss something?

I do agree that keeping code line lengths to a predictable maximum aids development, and a line length of 80 is certainly a time-honored standard.  If it is not documented in develop.txt it probably should be added there.

This is probably not the place for this observation, but at the end of develop.txt, in section 4.  Assumptions, it says "long is  32 or 64 bits signed, can hold a pointer".  Compilers targeting 64-bit Windows unfortunately do not all agree on the size of long, and long double.  On some long is 32-bits, on others 64-bits.  On some long double is 64-bits, on others 128 bits.  This means that a long cannot necessarily hold a pointer, which is always 64-bits.

Finally, a quick check of all the .c files shows:

```
    46 have a maximum line length < 80
    13 have a maximum line length = 80
  104 have a maximum line length > 80
```

The longest line has length 681, line 1997 of charset.c

I realize that retroactively imposing updated coding standards on a large codebase can often do more harm than good.  Requiring new code to adhere to them is just good practice.  I always make it a point to read and follow any and all documented coding styles and standards for a project, and if none are documented, do my best to follow the patterns of the existing code.  As this is my first direct submission to Vim I apologize if I missed something.

Note to self:  use a maximum line length of 80 in all future submissions to Vim.  😊

Reply:
Thanks for contributing and for even reading through develop.txt. We should probably update it for the things you mention (PR welcome ;) ). Note the current codestyle is currently under discussion, see #15789 since it is hard to enforce using a code-formatter. But changing it is not a nice process (reviewing it is an exhaustive excercise) and breaks e.g. git blame. Still I am thinking to make a cut and have it changed to make ours and contributors life in the future easier.

Reply:
Oh and most things mentions in develop.txt are *shoulds*  and not hard requirements, I think.

Reply:
"Breaks git blame" is a bit harsh: whitespace changes are ignored with `git blame -w`, for example, and some refactorings can be seen through with `-CCC -M` (what I've started calling "blame harder"). It's also possible to collect a list of "ignore-able blame revisions" for use with `--ignore-rev`.